sklearn/cross_validation.py

```diff
@@ -577,6 +578,150 @@ def __len__(self):
         return self.n_folds
 
 
+class BinnedStratifiedKFold(_BaseKFold):
```

Comment:
This needs to be removed from `cross_validation.py`. As we are deprecating the whole import path. You can safely have it implemented inside `model_selection` alone.